examples/e01_hello_world.c

```diff
@@ -7,47 +7,61 @@
 
 retcode_t get_iota_node_info(iota_client_service_t *iota_client_service, get_node_info_res_t *node_response) {
     retcode_t ret = RC_ERROR;
+    // Connect to the node
     ret = iota_client_get_node_info(iota_client_service, node_response);
 
+    // Define variables to determine whether trit conversion succeeds
     trit_t trytes_out[NUM_TRYTES_HASH + 1];
     size_t trits_count = 0;
+    // If the node returned data, print it to the console
     if (ret == RC_OK) {
         printf("appName %s \n", node_response->app_name->data);
         printf("appVersion %s \n", node_response->app_version->data);
 
-        //Converting returned latest_milestone trits to tryte chars
-        //Read for more information: https://docs.iota.org/docs/iota-basics/0.1/references/tryte-alphabet
+        // Convert the returned trits to trytes
+        // For more information about trits and trytes, see the IOTA documentation portal: https://docs.iota.org/docs/getting-started/0.1/introduction/ternary
         trits_count = flex_trits_to_trytes(trytes_out, NUM_TRYTES_HASH,
-                                           node_response->latest_milestone, NUM_TRITS_HASH,
-                                           NUM_TRITS_HASH);
+                                        node_response->latest_milestone, NUM_TRITS_HASH,
+                                        NUM_TRITS_HASH);
         if (trits_count == 0) {
-            printf("trit converting failed\n");
+            printf("Failed to convert trits to trytes\n");
             goto done;
         }
+        // Empty this string: we don't need it anymore
         trytes_out[NUM_TRYTES_HASH] = '\0';
+
         printf("latestMilestone %s \n", trytes_out);
         printf("latestMilestoneIndex %u\n", (uint32_t) node_response->latest_milestone_index);
+        printf("latestSolidSubtangleMilestoneIndex %u\n", (uint32_t)
+        node_response->latest_solid_subtangle_milestone_index);
 
-        printf("milestoneStratIndex %u\n", node_response->milestone_start_index);
+        printf("milestoneStartIndex %u\n", node_response->milestone_start_index);
         printf("neighbors %d \n", node_response->neighbors);
         printf("packetsQueueSize %d \n", node_response->packets_queue_size);
-        printf("time %"
-        PRIu64
-        "\n", node_response->time);
         printf("tips %u \n", node_response->tips);
         printf("transactionsToRequest %u\n", node_response->transactions_to_request);
-    }else{
-        printf("Node info failed.");
+        printf("features: ");
+        size_t num_features = get_node_info_req_features_len(node_res);
```

Comment:
Variable `node_res` does not exist. Probably `node_response` is the correct one.